modules/installation-configuration-parameters.adoc

```diff
@@ -2736,107 +2723,81 @@ The list of datacenters must match the list of datacenters specified in the `vce
     failureDomains:
       topology:
         datastore:
+ifdef::vsphere[]
+|Specifies the path to a vSphere datastore that stores virtual machines files for a failure domain. You must apply the `datastore` role to the vSphere vCenter datastore location.
+endif::vsphere[]
+ifdef::agent[]
 |The path to the vSphere datastore that holds virtual machine files, templates, and ISO images.
-[IMPORTANT]
-====
-You can specify the path of any datastore that exists in a datastore cluster.
+
+*Important:* You can specify the path of any datastore that exists in a datastore cluster.
```

Comment:
@dfitzmau I'm not sure I understand why this callout was changed. It seems like the support statement and the possible data loss warrant the callout. Likewise, I couldn't find any precedence for this kind of change or formatting. I _think_ we should put this back in [IMPORTANT] formatting?

Reply:
I wasn't sure why the admonition was dropped until I looked at the rest of the assembly. There were other instances of *Important*. I am thinking we are trying to get away from admonitions in tables, as they are [quite ungainly](https://74339--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_with_agent_based_installer/installation-config-parameters-agent#installation-configuration-parameters-optional_installation-config-parameters-agent).

Reply:
Hi Michael. Yes, this is correct.